driver_test.go

```diff
@@ -148,29 +148,18 @@ func runTests(t *testing.T, dsn string, tests ...func(dbt *DBTest)) {
 		defer db2.Close()
 	}
 
-	dsn3 := dsn + "&multiStatements=true"
```

Comment:
I removed this because tests using multistatement uses `runTestsWithMultiStatement()`.